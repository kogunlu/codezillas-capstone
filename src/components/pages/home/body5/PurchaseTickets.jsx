import React from "react";
import Card from "./Card";

function Purchase(){
    return(
<div>
<p className='text-4xl font-normal pt-14 pb-6 pl-40'>PURCHASE TICKETS</p>
<p className='text-l font-normal pb-16 pl-40'>PURCHASE TICKETS THAT CAN BE USED TO BOOK APPOINTMENTS!</p>

<div className="p-10 grid grid-cols-1 sm:grid-cols-1 md:grid-cols-3 lg:grid-cols-3 xl:grid-cols-3 gap-5">

        <div className=''>
        <Card title="5 TICKETS" dollar="10$" 
        button="PURCHASE"/>
        </div>
        <div className=''>
        <Card title="25 TICKETS" dollar="40$"
        button="PURCHASE"/>
        </div>
        <div className=''>
        <Card title="50 TICKETS" dollar="70$"
        button="PURCHASE"/>
        </div>
</div>
        <div className="pb-24 grid grid-cols-1 place-items-center ">
        <Card title="ARE YOU A COUNSELOR?" text="interested in joining our mental health platform? you decide your schedule and how much you want to work, we'll take care of the client referrals and billing details!" 
        button="LEARN MORE"/>
        </div>
    
</div>
        
    )
}
        
export default Purchase
