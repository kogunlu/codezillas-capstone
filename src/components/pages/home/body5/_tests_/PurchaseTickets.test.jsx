import React from "react";
import renderer from "react-test-renderer";
import PurchaseTickets from "../PurchaseTickets";


it('Renders correctly for PurchaseTickets Component', () => {
    const tree = renderer
    .create(<PurchaseTickets />)
    .toJSON();
    expect(tree).toMatchSnapshot();
  });
