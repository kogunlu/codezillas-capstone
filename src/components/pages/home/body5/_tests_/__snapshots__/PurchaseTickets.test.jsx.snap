// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly for PurchaseTickets Component 1`] = `
<div
  className="flex justify-center"
>
  <div
    className="flex flex-col items-center"
  >
    <div
      className="w-5/6 flex flex-col justify-between items-start"
    >
      <p
        className="text-4xl mt-12 mb-6 text-center lg:text-start 2xl:text-6xl font-bold "
      >
        PURCHASE TICKETS
      </p>
      <p
        className="text-l font-normal mb-16 text-center lg:text-start 2xl:text-3xl "
      >
        PURCHASE TICKETS THAT CAN BE USED TO BOOK APPOINTMENTS!
      </p>
    </div>
    <div
      className="w-5/6 flex flex-col gap-4 2xl:gap-20"
    >
      <div
        className="flex flex-wrap justify-center lg:justify-between w-full gap-4 "
      >
        <div
          className="border border-solid w-72 lg:w-96 border-gray-200 rounded-lg shadow-xl"
        >
          <div
            className="flex flex-col m-6"
          >
            <h1
              className="text-3xl font-semibold pt-12  flex flex-col items-center justify-center 2xl:text-5xl"
            >
              5 TICKETS
            </h1>
            <h2
              className="text-gray-400 text-xl pt-3 font-medium flex flex-col items-center justify-center 2xl:text-3xl 2xl:py-8"
            >
              10$
            </h2>
            <h2
              className="text-base pt-2 font-normal 2xl:text-2xl pb-6"
            />
            <div
              className="flex flex-col items-center justify-center pb-10 2xl:text-3xl 2xl:pb-12"
            >
              <input
                className="py-2 px-7 hover:bg-cyan-200 bg-cyan-400 font-medium rounded"
                type="button"
                value="PURCHASE"
              />
            </div>
          </div>
        </div>
        <div
          className="border border-solid w-72 lg:w-96 border-gray-200 rounded-lg shadow-xl"
        >
          <div
            className="flex flex-col m-6"
          >
            <h1
              className="text-3xl font-semibold pt-12  flex flex-col items-center justify-center 2xl:text-5xl"
            >
              25 TICKETS
            </h1>
            <h2
              className="text-gray-400 text-xl pt-3 font-medium flex flex-col items-center justify-center 2xl:text-3xl 2xl:py-8"
            >
              40$
            </h2>
            <h2
              className="text-base pt-2 font-normal 2xl:text-2xl pb-6"
            />
            <div
              className="flex flex-col items-center justify-center pb-10 2xl:text-3xl 2xl:pb-12"
            >
              <input
                className="py-2 px-7 hover:bg-cyan-200 bg-cyan-400 font-medium rounded"
                type="button"
                value="PURCHASE"
              />
            </div>
          </div>
        </div>
        <div
          className="border border-solid w-72 lg:w-96 border-gray-200 rounded-lg shadow-xl"
        >
          <div
            className="flex flex-col m-6"
          >
            <h1
              className="text-3xl font-semibold pt-12  flex flex-col items-center justify-center 2xl:text-5xl"
            >
              50 TICKETS
            </h1>
            <h2
              className="text-gray-400 text-xl pt-3 font-medium flex flex-col items-center justify-center 2xl:text-3xl 2xl:py-8"
            >
              70$
            </h2>
            <h2
              className="text-base pt-2 font-normal 2xl:text-2xl pb-6"
            />
            <div
              className="flex flex-col items-center justify-center pb-10 2xl:text-3xl 2xl:pb-12"
            >
              <input
                className="py-2 px-7 hover:bg-cyan-200 bg-cyan-400 font-medium rounded"
                type="button"
                value="PURCHASE"
              />
            </div>
          </div>
        </div>
      </div>
      <div
        className="flex justify-center mb-10"
      >
        <div
          className="border border-solid text-center w-full border-gray-200 rounded-lg shadow-xl"
        >
          <div
            className="flex flex-col m-6"
          >
            <h1
              className="text-3xl font-semibold pt-12  flex flex-col items-center justify-center 2xl:text-5xl"
            >
              ARE YOU A COUNSELOR?
            </h1>
            <h2
              className="text-gray-400 text-xl pt-3 font-medium flex flex-col items-center justify-center 2xl:text-3xl 2xl:py-8"
            />
            <h2
              className="text-base pt-2 font-normal 2xl:text-2xl pb-6"
            >
              interested in joining our mental health platform? you decide your schedule and how much you want to work, we'll take care of the client referrals and billing details!
            </h2>
            <div
              className="flex flex-col items-center justify-center pb-10 2xl:text-3xl 2xl:pb-12"
            >
              <input
                className="py-2 px-7 hover:bg-cyan-200 bg-cyan-400 font-medium rounded"
                type="button"
                value="LEARN MORE"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
